charts/steward/README.md

```diff
@@ -156,10 +156,26 @@ Common parameters:
+| <code>pipelineRuns.<wbr/>networkPolicies</code> | (map[string]string)<br/> The network policies selectable as network profiles in pipeline run specs. The key can be any valid YAML key not starting with underscore (`_`). The value must be a string containing a complete `networkpolicy.networking.k8s.io` resource manifest in YAML format. The `.metadata` section of the manifest can be omitted, as it will be replaced anyway. See the [Kubernetes documentation of network policies][k8s-networkpolicies] for details about Kubernetes network policies.<br/><br/> Note that Steward ensures that all pods in pipeline run namespaces are _isolated_ in terms of network policies. The policy defined here _adds_ egress and/or ingress rules. | A single entry named `default` whose value is a network policy defining rules that allow ingress traffic from all pods in the same namespace and egress traffic to the internet, the cluster DNS resolver and the Kubernetes API server. |
 | <code>pipelineRuns.<wbr/>limitRange</code> | (string)<br/> The limit range to be created in every pipeline run namespace. The value must be a string containing a complete `limitrange` resource manifest in YAML format. The `.metadata` section of the manifest can be omitted, as it will be replaced anyway. See the [Kubernetes documentation of limit ranges][k8s-limitranges] for details about Kubernetes limit ranges. | A limit range defining a default CPU request of 0.5 CPUs, a default CPU limit of 3 CPUs, a default memory request of 0.5 GiB and a default memory limit of 3 GiB.<br/><br/>This default limit range might change with newer releases of Steward. It is recommended to set an own limit range to avoid unexpected changes with Steward upgrades. |
 | <code>pipelineRuns.<wbr/>resourceQuota</code> | (string)<br/> The resource quota to be created in every pipeline run namespace. The value must be a string containing a complete `resourcequotas` resource manifest in YAML format. The `.metadata` section of the manifest can be omitted, as it will be replaced anyway. See the [Kubernetes documentation of resource quotas][k8s-resourcequotas] for details about Kubernetes resource quotas.| none |
 
+### Feature Flags
+
+| Parameter | Description | Default |
+|---|---|---|
+| <code>featureFlags</code> | (string)<br/> Feature flag definition.  | empty |
+
+The feature flags definition is a string with one or more feature flags. Multiple feature flags are separated by komma `,` or space `' '`.
+A feature flag can be set to enabled with adding the name optionally prefixed with a plus `+`.
+A feature flag can be set to disabled by adding the name prefixed with a dash `-`.
+
+| Feature Flag | Description | Default |
+| --- | --- | --- |
+| RetryOnInvalidPipelineRunsConfig | If enabled: Retries a pipeline run on a configuration error until the configuration error if fixed or the timeout is reached.</br>If disabled: The pipeline run is finished with an infrastructure error. | disabled |
```

Comment:
```suggestion
| RetryOnInvalidPipelineRunsConfig | If enabled, the pipeline run controller retries reconciling PipelineRun objects in case the controller configuration (in ConfigMaps) is invalid or cannot be loaded. It is assumed that the condition can be detected by a monitoring tool, triggers an alert and operators fix the issue in a timely manner. By that operator errors do not immediately break user pipeline runs. However, processing of PipelineRun objects may be delayed significantly in case of invalid configuration.<br/><br/> If disabled, the current behavior is used: immediately set all unfinished PipelineRun objects to finished with result code `error_infra`.<br/><br/>  The new behavior is supposed to become the default in a future release of Steward. | disabled |
```